openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -589,7 +589,7 @@ def _handle_duplicate_email_username(self, request, data):
                 accounts_settings.USERNAME_CONFLICT_MSG.format(username=username)
             )
             errors['username'] = [{'user_message': error_message}]
-            errors['username_suggestions'] = generate_username_suggestions(username)
+            errors['username_suggestions'] = generate_username_suggestions(data.get('name'))
```

Comment:
In my opinion, we should pass `username` as an argument because at this point form data would have a value in `username`, and would be better to suggest similar usernames